ArduPlane/quadplane.cpp

```diff
@@ -1903,10 +1903,7 @@ void QuadPlane::update_transition(void)
         // in half the transition time
         float transition_rate = tailsitter.transition_angle / float(transition_time_ms/2);
         uint32_t dt = now - transition_start_ms;
-        float pitch_cd;
-        pitch_cd = constrain_float((-transition_rate * dt)*100, -8500, 0);
-        // if already pitched forward at start of transition, wait until curve catches up
-        plane.nav_pitch_cd = (pitch_cd > transition_initial_pitch)? transition_initial_pitch : pitch_cd;
+        plane.nav_pitch_cd = constrain_float(transition_initial_pitch - (transition_rate * dt)*100, -8500, 8500);
```

Comment:
don't you need to take initial pitch into account when calculating the transition rate here?

Reply:
sorry, I don't follow?

Reply:
Doesn't the transition_rate calculation assume initial pitch is zero?

Reply:
We have the true initial pitch in the calculation,`transition_initial_pitch`